vagrant/config/versions.yaml

```diff
@@ -29,8 +11,8 @@ installers:
       - 'ubuntu1804'
 
   - foreman: '1.20'
-    katello: '3.9'
```

Comment:
The same goes for Katello 3.11: it needs Katello 3.9 in upgrades.

Reply:
restored 3.9 too